### PR TITLE
feat: add database cert alert

### PR DESCRIPTION
**Pull request by ahmaxed**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

This pr adds an alert to the top of the relational database certification page with a link to the related article.

### CODE REVIEW

**Comment by raisedadead** [comment]
This PR is valuable. 

I have temporarily targeted it to a temporary `next` branch with Tom's work that this depends on. We can finalize the details later, but I wanted to closely look at the differences. 

For now, #45347 will revert #45169

File: client/src/utils/is-a-cert.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { SuperBlocks } from '../../../config/certification-settings';
+
+export function IsNewRespCert(superBlock: string): boolean {
```
Comment (ojeytonwilliams) [inline client/src/utils/is-a-cert.ts:3]:
```suggestion
export function isNewRespCert(superBlock: string): boolean {
```
Nitpick

File: client/src/utils/is-a-cert.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+  return superBlock === SuperBlocks.RespWebDesignNew;
+}
+
+export function IsRelationalDbCert(superBlock: string): boolean {
```
Comment (ojeytonwilliams) [inline client/src/utils/is-a-cert.ts:7]:
```suggestion
export function isRelationalDbCert(superBlock: string): boolean {
```

**Review by ojeytonwilliams — CHANGES_REQUESTED** [review]

**Review by ojeytonwilliams — DISMISSED** [review]
LGTM.

It doesn't seem worth it to split these tiny functions off into their own file, but the functionality is good.

~Is the plan to merge this after publishing the news article?~ Nevermind, I see that it's going into the next branch!

**Comment by moT01** [comment]
I think we want to go with a forum post here. I can create it and share a link.

**Comment by raisedadead** [comment]
Thanks @moT01 

Once you have done that, we can get this in once we have your PR recreated to `main` followed by re-targeting this PR to `main` as well.

**Comment by moT01** [comment]
I made the forum post and pushed the link to this PR

> Once you have done that, we can get this in once we have your PR recreated to main followed by re-targeting this PR to main as well.

Can you explain the process a little more to me @raisedadead? I'm not sure I understand. I assumed we would merge this to next, merge my security issue fix/webhook PR to next, then merge a PR of next into main or something. Those are the only two PR's that should be needed.

Do we just want to re-target both of them to main? I think that's what you meant - looks like maybe just a typo there that confused me. I'm fine with it either way.

**Comment by raisedadead** [comment]
> Do we just want to re-target both of them to main? I think that's what you meant

Yes - that's exactly the plan. We can start by landing the security fix PR first on `main`

File: client/i18n/locales/english/intro.json [hunk 3]
```diff
@@ -766,6 +766,7 @@
     "legacy-header": "Legacy Courses",
     "legacy-desc": "These courses are no longer part of the certification path, but are still available for you to further your learning.",
     "viewing-upcoming-change": "You are looking at a beta page. ",
-    "go-back-to-learn": "Go back to the stable version of the curriculum."
+    "go-back-to-learn": "Go back to the stable version of the curriculum.",
+    "read-database-cert-article": "Please read this article before proceeding."
```
Comment (nhcarrigan) [inline client/i18n/locales/english/intro.json:770]: Nitpick, but do we want to call it "article" or "forum post"?

**Review by naomi-lgbt — DISMISSED** [review]
This looks good to me, aside from one nitpick question.

**Review by naomi-lgbt — APPROVED** [review]
Nitpick resolved, now this looks gooder.

**Review by ojeytonwilliams — APPROVED** [review]
LGTM :+1:

**Pull request merged**